packages/web-components/src/components/va-table/va-table.tsx

```diff
@@ -37,6 +39,12 @@ export class VaTable {
    */
   @Prop() stacked?: boolean = true;
   
+
+  /**
+   * Is the table sortable
+   */
+  @Prop() sortable: boolean = false; 
```

Comment:
Have we tested how this behaves with [pagination](https://sortable-table--65a6e2ed2314f7b8f98609d8.chromatic.com/?path=/docs/uswds-va-table--docs#with-pagination)?

Reply:
@jamigibbs there was a bug (not actually related to pagination but revealed in the pagination example) that is now fixed